cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/ZookeeperLeaderFinderTest.java

```diff
@@ -190,9 +193,12 @@ public void stop() {
     }
 
     @AfterEach
-    public void stopZks() {
+    public void stopZks(VertxTestContext vertxTestContext) {
+        if (zks.isEmpty()) {
+            vertxTestContext.completeNow();
+        }
         for (FakeZk zk : zks) {
-            zk.stop();
+            zk.stop(vertxTestContext);
         }
```

Comment:
It seems a bit weird/ugly to me that the text context can be completed in two different ways, one via `completeNow()`, if `zks` is empty, and the other by the checkpoints getting flagged when zks is nonempty. Is there a simple way we can just make the call to `zk.stop()` be synchronous? That way you could just call `completeNow()` after the loop, no matter the emptiness of `zks`.